CONTRIBUTING.md

```diff
@@ -106,6 +107,7 @@ A short list of tools used in `dbt-core` testing that will be helpful to your un
 - [`pytest`](https://docs.pytest.org/en/latest/) to discover/run tests
 - [`make`](https://users.cs.duke.edu/~ola/courses/programming/Makefiles/Makefiles.html) - but don't worry too much, nobody _really_ understands how make works and our Makefile is super simple
 - [`flake8`](https://flake8.pycqa.org/en/latest/) for code linting
+- [`black`](https://github.com/psf/black) for code formatting
```

Comment:
do we need black included in some kind of dev-requirements file or will pre-hooks handle that installation?